db_dtypes/__init__.py

```diff
@@ -47,15 +46,6 @@
 _NP_BOX_DTYPE = "datetime64[us]"
 
 
-# To use JSONArray and JSONDtype, you'll need Pandas 1.5.0 or later. With the removal
-# of Python 3.7 compatibility, the minimum Pandas version will be updated to 1.5.0.
-if packaging.version.Version(pandas.__version__) >= packaging.version.Version("1.5.0"):
```

Comment:
From the comments, it seems removing Python 3.7 could ensure pandas >= 1.5.0, and thus ensure JSONArray and JSONDtype to be available?

Reply:
Removed.